modules/core/config/MobileConfig.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+/**
+ * Model for mobile backend configuration
+ */
+public struct MobileConfig: Codable {
+    // Service namespace
+    public var namespace: String?
+    // List of services
+    public var services: [MobileService]?
```

Comment:
In case of arrays it better to create instance of it like `public var services = [MobileService]()`.